content/copilot/index.md

```diff
@@ -0,0 +1,168 @@
+---
```

Comment:
General comment - I couldn't see where this comment was linked from in the preview of the docs. 

Where is it linked from :) ?

Reply:
I've added a navigation entry for this page - I wasn't sure if we wanted to have it listed in the navigation, or not. I also removed the `sitemap: false` metadata, which means that this page will be indexed by search engines and searchable.